### ISSUE TITLE
Error installing Annotald

### ISSUE THREAD

**Issue by mfarasyn** [Annotald/annotald#88]
I have a problem installing Annotald on my new distribution (Ubuntu 16.04). Both Python 2 and Python 3 are installed and pip 9.0.1 is installed for Python 2. I tried several commands (sudo -H pip install annotald, sudo -H pip2 install annotald, sudo python 2.7 -m pip install annotald), but they all give me the error below. What could be the reason for that? 

```
Collecting Annotald
  Using cached annotald-1.3.8-py2.py3-none-any.whl
Requirement already satisfied: argparse in /usr/lib/python2.7 (from Annotald)
Requirement already satisfied: cherrypy in /usr/lib/python2.7/dist-packages (from Annotald)
Collecting nltk==2.0.5 (from Annotald)
  Using cached nltk-2.0.5.tar.gz
    Complete output from command python setup.py egg_info:
    Downloading http://pypi.python.org/packages/source/d/distribute/distribute-0.6.21.tar.gz
    Extracting in /tmp/tmpCRfA6K
    Now working in /tmp/tmpCRfA6K/distribute-0.6.21
    Building a Distribute egg in /tmp/pip-build-tcErbo/nltk
    Traceback (most recent call last):
      File "setup.py", line 37, in <module>
        exec(open(init_path).read(), d)
      File "<string>", line 8, in <module>
      File "/tmp/tmpCRfA6K/distribute-0.6.21/setuptools/__init__.py", line 2, in <module>
        from setuptools.extension import Extension, Library
      File "/tmp/tmpCRfA6K/distribute-0.6.21/setuptools/extension.py", line 2, in <module>
        from setuptools.dist import _get_unpatched
      File "/tmp/tmpCRfA6K/distribute-0.6.21/setuptools/dist.py", line 6, in <module>
        from setuptools.command.install import install
      File "/tmp/tmpCRfA6K/distribute-0.6.21/setuptools/command/__init__.py", line 8, in <module>
        from setuptools.command import install_scripts
      File "/tmp/tmpCRfA6K/distribute-0.6.21/setuptools/command/install_scripts.py", line 3, in <module>
        from pkg_resources import Distribution, PathMetadata, ensure_directory
      File "/tmp/tmpCRfA6K/distribute-0.6.21/pkg_resources.py", line 2729, in <module>
        add_activation_listener(lambda dist: dist.activate())
      File "/tmp/tmpCRfA6K/distribute-0.6.21/pkg_resources.py", line 700, in subscribe
        callback(dist)
      File "/tmp/tmpCRfA6K/distribute-0.6.21/pkg_resources.py", line 2729, in <lambda>
        add_activation_listener(lambda dist: dist.activate())
      File "/tmp/tmpCRfA6K/distribute-0.6.21/pkg_resources.py", line 2229, in activate
        self.insert_on(path)
      File "/tmp/tmpCRfA6K/distribute-0.6.21/pkg_resources.py", line 2330, in insert_on
        "with distribute. Found one at %s" % str(self.location))
    ValueError: A 0.7-series setuptools cannot be installed with distribute. Found one at /usr/lib/python2.7/dist-packages
    /tmp/pip-build-tcErbo/nltk/distribute-0.6.21-py2.7.egg
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-tcErbo/nltk/setup.py", line 23, in <module>
        distribute_setup.use_setuptools()
      File "/tmp/pip-build-tcErbo/nltk/distribute_setup.py", line 145, in use_setuptools
        return _do_download(version, download_base, to_dir, download_delay)
      File "/tmp/pip-build-tcErbo/nltk/distribute_setup.py", line 125, in _do_download
        _build_egg(egg, tarball, to_dir)
      File "/tmp/pip-build-tcErbo/nltk/distribute_setup.py", line 116, in _build_egg
        raise IOError('Could not build the egg.')
    IOError: Could not build the egg.
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-tcErbo/nltk/
```

**Comment by aecay** [comment]
(Sorry it has taken me a while to get back to this)

Can you try running the following command: `pip install --user --update setuptools`?  Then try installing annotald again.  If that does not work, can you report back with the output of `pip list`?

**Comment by pfaria** [comment]
On Mac, I've come to the following solution:

General:

- Download pip: curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
- Install 'pip': sudo python get-pip.py
- Install 'virtualenv': sudo pip install virtualenv

ANNOTALD:

- Create virtual environment: virtualenv annotald
- Activate it: source annotald/bin/activate

- Download nltk 2.0.5: https://files.pythonhosted.org/packages/50/9e/39418026bf8013bbc2852c7aec3fb21e4339f6cd694934496d67a19b53b8/nltk-2.0.5.tar.gz
- sudo pip install distribute
- sudo pip install nose tornado
- sudo pip install PyYAML
- sudo pip install six==1.11.0 --ignore-installed
- NLTK [in its unpacked folder]:
  - Open 'setup.py' and comment (with a '#') the lines 'import distribute_setup' and 'distribute_setup.use_setuptools()'
  - sudo python setup.py install
- sudo pip install annotald

Work sessions:

- source annotald/bin/activate
- Run Annotald as needed
- deactivate

**Comment by aecay** [comment]
I believe these problems are fixed by the new version 1.3.10 of Annotald.
